workspace/run_sft_ft_audio_lm.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
let's refrain from adding more files to workspace since this branch will be merged into main later. can include these two files in other branch 

can move this under example tho

Reply:
let's move the py scripts to examples in the next refactoring PR